_posts/2020-05-27-cockpit-220.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 220
+author: marusak
+date: 2020-05-27
+tags: cockpit linux
+slug: cockpit-220
+category: release
+summary: Cockpit with new navigation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 220.
+
+### New navigation with integrated hosts switching
+
+The navigation has been redesigned and brings four major improvements:
+- **One level navigation**: The current two level navigation has been squashed for better use of space and better discoverability
+- **Integrated hosts switching**: Switching between hosts as well as editing them can be done now directly from the navigation without using the 'Dashboard' component
+- **Better discoverability of applications**: Applications are shown as the first group in the menu and are also searchable
+- **Super indicator stuff**: Yeah, lets mention #14055 here
+
+![New one level navigation](/images/one-level-nav.png)
+![Integrated host switcher](/images/host-switcher.png)
+
+### Logs: Inline help for filtering
+
+In the [previous release](https://cockpit-project.org/blog/cockpit-219.html) new possibilities how to search for logs were introduced.
+Now is also provided inline overview of accepted options and `journalctl` command which shows the same log entries in terminal.
```

Comment:
This release adds a help button that shows an overview of accepted options, and the `journalctl` command corresponding to the current filter.